### ISSUE TITLE
Anamnesis Scaling / Overworld Camera

### ISSUE THREAD

**Issue by OzenScythe** [imchillin/Anamnesis#912]
1. Scaling. I am not able to Scale a multitude of people while IN G-POSE. I normally have to have CMT open with Anamnesis to scale multiple people (not on my friend's list or party) at the same time as I will get that Anamnesis Message covering the scaling feature saying I need to be outside of G-pose to make changes. CMT has no problem multiple people not on my friend's list or random NPCs etc.

2. Overrworld Camera.  I used to be able to tweak the FOV of my actual game without going into G-pose with CMT. Anamnesis does not have these tools online at the moment and would like to see it return.

3. REQUESTING A NEW FEATURE: I am not sure if this is possible. But if this could be added, this would help people who are into more Video Work. The ability to not only tilt up and down, but tilt camera Side to Side while IN-GAME. This can be done in G-Pose, but models will not be in REAL TIME MOTION unless they are not in a party / not on your friend's list. (That's the work around to this solution in the meantime) but it be nice to do this on command in base game if possible so if I had actors who are my friends, i wouldn't have to unfriend them constantly if I wanted to use Anamnesis for more Video Work. If not possible, I will stick to the work around.

Thank you again to your team for providing game changing tools to the creative community. I can't live without these tools and I only want to see them get even better. Thank you for your hardwork and to give people the ability to turn their visions into reality.

Ozen Scythe, Gilgamesh.

**Comment by Peebs-miqo** [comment]
I can't help in regards to the scaling issue, but I can say the developers of Anamnesis have no intention of allowing camera controls outside of GPose. Allowing camera controls outside of GPose, in our view, facilitates unfair game advantages that we do not want. This is covered in the 'Before Making a Suggestion' section in the Anamnesis Wiki.

**Comment by Ani-ki** [comment]
Re: scaling- Just to confirm, you're using the scaling option on the pose tab and not the one on the character tab, correct?

**Comment by OzenScythe** [comment]
> I can't help in regards to the scaling issue, but I can say the developers of Anamnesis have no intention of allowing camera controls outside of GPose. Allowing camera controls outside of GPose, in our view, facilitates unfair game advantages that we do not want. This is covered in the 'Before Making a Suggestion' section in the Anamnesis Wiki.

It still exist on CMT to alter overworld FOV to provide my view of the battlefield. If that was the case, it wouldn't be there to begin with.

**Comment by Ani-ki** [comment]
> > I can't help in regards to the scaling issue, but I can say the developers of Anamnesis have no intention of allowing camera controls outside of GPose. Allowing camera controls outside of GPose, in our view, facilitates unfair game advantages that we do not want. This is covered in the 'Before Making a Suggestion' section in the Anamnesis Wiki.
> 
> It still exist on CMT to alter overworld FOV to provide my view of the battlefield. If that was the case, it wouldn't be there to begin with.

Anamnesis =/= CMTool.

**Comment by OzenScythe** [comment]
> 

Correct. I had a project where I had to use about 5 PCs / 5 NPCs. The NPCs since they are not apart of my party, needed to utilize scaling from CMT on the Character Tab. Pose/ Character Tab on Anamnesis does not work with other entities such as NPCs as you will be blocked by a message saying you need to be outside of G-Pose.

**Comment by Ani-ki** [comment]
This is because **you aren't using the scale option in the Pose tab**, as I said in my first comment.

![image](https://user-images.githubusercontent.com/56663405/161545849-dcf7b969-f4de-44aa-860c-9f6cb69ea298.png)

**Comment by Peebs-miqo** [comment]
1. Is the entire Model Panel disappearing, or are you seeing an error message? If you are seeing an error message, what was the exact wording?
2. Can you clarify if this scaling issue is only applying to unnamed NPCs? Or is it also applying to Named Actors?

We've not come across anything like this before, if it's happening as you described, and without a way to replicate we can't really help.
